app/cells/decidim/conferences/conference_m/tags.erb

```diff
@@ -0,0 +1,24 @@
+<div class="card__block">
+    <div class="card__icondata">
+        <ul class="card-data">
+            <li class="card-data__item">
+                <%= icon "datetime", class: "icon--big", role: "img", aria_label: t("decidim.meetings.meetings.show.date") %>
+            </li>
+            <li class="card-data__item">
+                <div class="card-data__item—multiple">
+                    <div>
+                        <strong>
+                        <%= model.start_time %>
```

Comment:
```suggestion
                        <%= l model.start_time, format: :decidim_with_month_name %>
```

Reply:
l is a shortcut to format dates, in decidim there's some standard formats (like the one suggested)

Reply:
Thanks. Done!